db/seeds.rb

```diff
@@ -1,7 +1,135 @@
-# This file should contain all the record creation needed to seed the database with its default values.
-# The data can then be loaded with the bin/rails db:seed command (or created alongside the database with db:setup).
-#
-# Examples:
-#
-#   movies = Movie.create([{ name: "Star Wars" }, { name: "Lord of the Rings" }])
-#   Character.create(name: "Luke", movie: movies.first)
+# Clear the database to prevent duplicate records
+Contact.destroy_all
+User.destroy_all
+Friendship.destroy_all
+Interaction.destroy_all
+Medium.destroy_all
+Note.destroy_all
+Reminder.destroy_all
+UserSetting.destroy_all
```

Comment:
We should get in the habit of ading a `dependent: destroy` on the models that have a `has_many`. This way we would only need a `User.destroy_all` line here avoiding redundant code.

Reply:
Not sure why there weren't any changes here if you both reacted with thumbs up.

Reply:
Done 🌈